tools/testbench/makefile-xcc

```diff
@@ -0,0 +1,127 @@
+# Build xt-run version of testbench
+
+CC = xt-xcc
+LD = xt-xc++
+OBJDUMP = xt-objdump
+OPTFLAGS = -std=gnu99 -fgnu89-inline -g -O2 -Wall -Wmissing-prototypes \
+	 -Wno-pointer-to-int-cast -Wno-int-to-pointer-cast \
+	 -Wpointer-arith -Werror
+INSTALLPATH = build_testbench/install/bin
+INCLUDEPATHS = -I./include -I../tplg_parser/include \
+	     -I../../src/arch/host/include \
+	     -I../../src/platform/library/include \
+	     -I../../src/include \
+	     -I../../xtos/include
+DEFINES = -DCONFIG_IPC_MAJOR_3 -DCONFIG_CORE_COUNT=1 \
```

Comment:
How to understand this xt-testbench work scope? only support IPC3? since some of our code have ipc3 and ipc4 support, if testbench only support ipc3, this maybe a problem.

Reply:
Yep, we will need IPC4. We have a testing gap while this is IPC3 only.

Reply:
ok